src/components/Calculator.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import '../index.css';
+
+function Calculator() {
+  return (
+    <div className="calculator">
+      <div className="calculator__view__container">
+        <div className="calculator__view">
+          <h1 className="calculator__view__h1">0</h1>
+        </div>
+      </div>
+      <div className="calculator__rows">
+        <div className="calculator__row">
```

Comment:
- [ ] I suggest using a class-based component instead of functional for your Calculator.js because it is mandatory for this milestone.